src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -108,12 +107,11 @@
     (i18n/label :t/request-to-join-community)))
 
 (defn join-community
-  [{:keys [joined can-join? requested-to-join-at
+  [{:keys [joined can-join?
            community-color permissions]
-    :as   community}]
-  (let [pending?      (pos? requested-to-join-at)
-        is-open?      (not= constants/community-channel-access-on-request (:access permissions))
-        node-offline? (and can-join? (not joined) (pos? requested-to-join-at))]
+    :as   community} pending?]
+  (let [is-open?      (not= constants/community-channel-access-on-request (:access permissions))
```

Comment:
This is a bit fragile, maybe is better to positively match it? 
```(= constants/community-channel-access-no-membership (:access permissions)```
or 
if there's multiple open communities, something like:
```
(def open-communities #{type-a type-b}) ;; defined somewhere else
(let [is-open? (open-communities (:access permissions)]
...)
```

Reply:
Will adjust